python/ray/serve/tests/test_standalone2.py

```diff
@@ -1204,6 +1205,56 @@ async def waiter(*args):
         serve.shutdown()
 
 
+@pytest.mark.parametrize(
+    "ray_instance",
+    [
+        {
+            "LISTEN_FOR_CHANGE_REQUEST_TIMEOUT_S_LOWER_BOUND": "1",
+            "LISTEN_FOR_CHANGE_REQUEST_TIMEOUT_S_UPPER_BOUND": "2",
+        },
+    ],
+    indirect=True,
+)
+def test_long_poll_timeout_with_max_concurrent_queries(ray_instance):
```

Comment:
Can you help me understand how and why this test case fails before the issue is fixed?

Reply:
I see, can we add a unit test for `compute_iterable_delta` that uses the `RunningReplicaInfo` object and check it gives the expected results?

Reply:
done!